apps/checkout-app/__tests__/pages/api/webhooks/mollie.test.ts

```diff
@@ -0,0 +1,34 @@
+import handler from "@/checkout-app/pages/api/webhooks/mollie";
+import {
+  mockRequest,
+  setupPollyMiddleware,
+  setupRecording,
+} from "@/checkout-app/test-utils";
+
+describe("/api/webhooks/mollie", () => {
+  const context = setupRecording();
+
+  beforeEach(() => {
+    setupPollyMiddleware(context.polly.server);
+  });
+
+  it("handles invalid requests payments", async () => {
```

Comment:
Maybe its over-verbose, but reading test I'm not exactly sure what is invalid.

Maybe doing `req.body = {}` will be more explicit, that `{}` equals "invalid" here?